cli/tests/integrations/test_marathon_pod.py

```diff
@@ -111,26 +113,63 @@ def _assert_pod_list_json_subset(expected_json, actual_json):
     for expected_pod in expected_json:
         pod_id = expected_pod['id']
         actual_pod = actual_pods_by_id[pod_id]
-        _assert_pod_json(expected_pod['spec'], actual_pod['spec'])
+        _assert_pod_spec_json(expected_pod['spec'], actual_pod['spec'])
 
     assert len(actual_json) == len(expected_json)
 
 
-def _assert_pod_json(expected_pod, actual_pod):
-    """Checks that the "actual" pod JSON matches the "expected" pod JSON.
+def _assert_pod_status_json(expected_pod_status, actual_pod_status):
+    """Checks that the "actual" pod status JSON matched the "expected" JSON.
 
     The comparison only looks at specific fields that are present in the
     test data used by this module.
 
-    :param expected_pod: contains the baseline values for the comparison
-    :type expected_pod: {}
-    :param actual_pod: has its fields checked against the expected fields
-    :type actual_pod: {}
+    :param expected_pod_status: contains the baseline values for the comparison
+    :type expected_pod_status: {}
+    :param actual_pod_status: has its fields checked against expected's fields
+    :type actual_pod_status: {}
     :rtype: None
     """
 
-    expected_containers = expected_pod['containers']
-    actual_containers = actual_pod['containers']
+    assert actual_pod_status['id'] == expected_pod_status['id']
+    assert actual_pod_status['status'] == expected_pod_status['status']
+    assert len(actual_pod_status['instances']) == \
+        len(expected_pod_status['instances'])
+
+    _assert_pod_spec_json(expected_pod_status['spec'],
+                          actual_pod_status['spec'])
+
+    for i in range(len(expected_pod_status['instances'])):
```

Comment:
I could change the "expected" file to have only one instance, and update the code to compare that instance to every instance in the "actual" JSON. But then the "expected" file won't mirror the structure of the "actual" JSON anymore. Is that okay?